manifests/server/options.pp

```diff
@@ -119,6 +127,19 @@
     fail("The zone_notify must be ${valid_zone_notify}")
   }
 
+  $valid_dnssec_validation = ['yes', 'no', 'auto', 'absent']
+  if $dnssec_validation != undef and !member($valid_dnssec_validation, $dnssec_validation) {
+    fail("The dnssec_validation must be ${valid_dnssec_validation}")
```

Comment:
Actually, I was following the pattern used several times earlier in the file.  I can change this to validate_re - but should I change the previous checks that look the same?

Example:

    if $check_names_master != undef and !member($valid_check_names, $check_names_master) {
      fail("The check names policy check_names_master must be ${valid_check_names}")
    }